src/sentry/static/sentry/app/actionCreators/serviceIncidents.jsx

```diff
@@ -27,31 +25,28 @@ function getIncidentsFromIncidentResponse(incidents) {
   return [log, isMajor ? 'major' : 'minor'];
 }
 
-export function load() {
+export async function load() {
   const cfg = ConfigStore.get('statuspage');
   if (cfg && cfg.id) {
     ServiceIncidentActions.update();
 
-    $.ajax({
-      type: 'GET',
-      url: 'https://' + cfg.id + '.' + cfg.api_host + '/api/v2/incidents/unresolved.json',
-      crossDomain: true,
-      cache: false,
-      success: data => {
-        const [incidents, indicator] = getIncidentsFromIncidentResponse(data.incidents);
-        ServiceIncidentActions.updateSuccess({
-          status: {
-            incidents,
-            indicator,
-            url: data.page.url,
-          },
-        });
-      },
-      error: () => {
-        ServiceIncidentActions.updateError({
-          status: null,
-        });
-      },
-    });
+    const response = await fetch(
```

Comment:
Do our babel polyfills include support for `fetch()` from a cursory look at google it didn't sound like the preset-env doesn't include a fetch fill.

Reply:
https://github.com/getsentry/sentry/pull/16005#issuecomment-563363242